### ISSUE TITLE
Changing the branch 'master' to 'main' broke zzet.rbenv

### ISSUE THREAD

**Issue by asfarley** [rbenv/rbenv-installer#43]
Hi,

It seems that the removal of the 'master' branch has broken the Ansible role zzet.rbenv. 

I am just logging an issue here in case anyone runs into the same thing. I can submit a pull request to fix zzet.rbenv, but it looks like this issue could re-occur any time the branches are renamed in rbenv. 

Furthermore, it looks like this repository (rbenv-installer) is unique in removing 'master' in favor of 'main' while the other repositories still have 'master'. I might suggest switching this one back to 'master' if it's not too inconvenient.

**Comment by asfarley** [comment]
Maybe I should phrase this as a question: what drove the branch name-change? Is there anything that can be done in the future to prevent this kind of breakage?

**Comment by asfarley** [comment]
My mistake, this was already fixed in zzet.rbenv.

**Comment by mislav** [comment]
@asfarley Thank you for reporting, and sorry for the trouble. After a default branch rename, GitHub transparently redirects a lot of requests for the old branch name to the new branch name, so I thought to try the branch rename feature in this repository before I attempt it in other `rbenv/*` repositories. However, after seeing so many things break as a consequence (thanks to reports like yours), I won't be renaming branches in other repositories. However, I will not rename the branch back in this repository either— I decided to bite the bullet and stay with the rename, and hope that people update their scripts. (I did not know that so many people relied on this repository for automation.)
